### PR TITLE
UI tweaks - date consistency, EPG picon background, text fixes

### PR DESCRIPTION
**Pull request by wedebe**
- Fixed `simulate` header text (`title` -> `match text`
- Dark theme `loading...` overlay text colour fix
- Soften channel name/picon header background
- Consistent, i10n-ified start/end dates

Screen Shot 2020-11-17 at 23 36 45
Screen Shot 2020-11-17 at 23 32 21

### CODE REVIEW

File: plugin/controllers/models/services.py [hunk 1]
```diff
@@ -728,6 +728,11 @@ def getEvent(ref, idev, encode=True):
 		info['genre'], info['genreid'] = convertGenre(event[8])
 		info['picon'] = getPicon(event[7])
 		info['timer'] = getTimerEventStatus(event, eventLookupTable)
+		info['formattedstrings'] = {
```
Comment (jbleyel) [inline plugin/controllers/models/services.py:731]: Datetime formating should be a client part and not part or the api.

File: plugin/controllers/models/services.py [hunk 2]
```diff
@@ -999,6 +1004,11 @@ def getSearchEpg(sstr, endtime=None, fulldesc=False, bouquetsonly=False, encode=
 			ev['picon'] = getPicon(event[7])
 			ev['now_timestamp'] = None
 			ev['genre'], ev['genreid'] = convertGenre(event[8])
+			ev['formattedstrings'] = {
```
Comment (jbleyel) [inline plugin/controllers/models/services.py:1007]: Datetime formating should be a client part and not part or the api.

File: plugin/controllers/models/timers.py [hunk 3]
```diff
@@ -177,7 +177,12 @@ def getTimers(session):
 			"pipzap": pipzap,
 			"isAutoTimer": isAutoTimer,
 			"allow_duplicate": allow_duplicate,
-			"autoadjust": autoadjust
+			"autoadjust": autoadjust,
+			"formattedstrings": {
```
Comment (jbleyel) [inline plugin/controllers/models/timers.py:181]: Datetime formating should be a client part and not part or the api.

File: plugin/controllers/models/timers.py [hunk 4]
```diff
@@ -227,7 +232,12 @@ def addTimer(session, serviceref, begin, end, name, description, disabled, justp
 					"begin": conflict.begin,
 					"end": conflict.end,
 					"realbegin": strftime("%d.%m.%Y %H:%M", (localtime(float(conflict.begin)))),
-					"realend": strftime("%d.%m.%Y %H:%M", (localtime(float(conflict.end))))
+					"realend": strftime("%d.%m.%Y %H:%M", (localtime(float(conflict.end)))),
+					"formattedstrings": {
```
Comment (jbleyel) [inline plugin/controllers/models/timers.py:236]: Datetime formating should be a client part and not part or the api.

File: plugin/controllers/models/timers.py [hunk 5]
```diff
@@ -381,7 +391,12 @@ def editTimer(session, serviceref, begin, end, name, description, disabled, just
 						"begin": conflict.begin,
 						"end": conflict.end,
 						"realbegin": strftime("%d.%m.%Y %H:%M", (localtime(float(conflict.begin)))),
-						"realend": strftime("%d.%m.%Y %H:%M", (localtime(float(conflict.end))))
+						"realend": strftime("%d.%m.%Y %H:%M", (localtime(float(conflict.end)))),
+						"formattedstrings": {
```
Comment (jbleyel) [inline plugin/controllers/models/timers.py:395]: Datetime formating should be a client part and not part or the api.

File: plugin/controllers/utilities.py [hunk 6]
```diff
@@ -347,6 +348,29 @@ def getEventInfoProvider(moviedb):
 	return providerData
 
 
+def getFormattedDateTime(timestamp):
```
Comment (jbleyel) [inline plugin/controllers/utilities.py:351]: unicode? Why? Do you have test PY3?

**Comment by jbleyel** [comment]
Human readable datetime format via api for a single event or for timers is ok to me.
For servicelist NOT.

**Comment by jbleyel** [comment]
The getFormattedXX functions are ok to me if you use these inside of the twisted templates.
But i think it's better in javascript.

You may have a performance issue if you do this inside of a large epg service list.
That's why i don't like this.

I hope you understand.

**Comment by wedebe** [comment]
I believe localisation should match the stb/receiver's format - which is why I made changes in python, after all, the user has set up their box to their desired language/region (otherwise the box's own gui would show its format, but the browser its own, which may well differ). There is, however, an existing dd.mm.yy format in some of the python code (there's also an _str version of a date, to add to the confusion!)

Agree with the likely performance impact on the EPG, I think storing a copy in the browser's localstorage would help significantly.

I'll come back to this another time.

**Pull request merged**